gdcdictionary/schemas/pathology_detail.yaml

```
@@ -451,6 +493,14 @@ properties:
     type: integer
     minimum: 0
 
+  lymph_nodes_removed:
+    $ref:
+      - "_terms.yaml#/lymph_nodes_removed/common
```

This should fix the build error:
```suggestion
      - "_terms.yaml#/lymph_nodes_removed/common"
```

---

Thanks @mpsolano!  Made the commit, didn't see that cool "Commit suggestion" feature until I was done.